docs/src/pages/[platform]/connected-components/authenticator/customization/formfields/form-fields.flutter.mdx

```diff
@@ -23,5 +23,13 @@ A list of the codes can also be found in the `DialCode` class.
   {({platform}) => import(`./dialcode/form-fields.flutter.mdx`)}
 </Fragment>
 
+### TOTP Issuer
+The TOTP issuer is the name that will be shown in TOTP applications preceding the account name. In most cases, this should be the name of your app. For example, if your app is called "My App", your user will see "My App" - "username" in their TOTP app. This can be customized by adding the `TotpOptions` argument to the Authenticator component with a value for `issuer`. 
+
+Note: Unless changed, the default issuer is the application name retrieved from the project configuration.
```

Comment:
can we specify that it is the name in the pubspec.yaml file and give the key name?